packages/components/src/lib/queries/lexical-query.js

```diff
@@ -362,87 +367,6 @@ export default class LexicalQuery extends Query {
     }
   }
 
-  async partialDataRetrieval () {
```

Comment:
This functionality has been split into three separate functions and was moved to the `Lexis`.